apps/src/music/progress/MusicConditions.ts

```diff
@@ -27,7 +27,13 @@ export const MusicConditions: ConditionNames = {
     name: 'played_sound_in_function',
     valueType: 'string',
     description:
-      'Checks if a sound is playing in a given function. Simple2 only. Ex. Value: chorus',
+      "Checks if a sound is playing in a given function.  The function is identified by name, which doesn't work if it' s localized. Simple2 only. Ex. Value: chorus",
```

Comment:
Should we go ahead and remove this validation type to avoid confusion?

Reply:
Good question.  It's currently used in 13 other levels, so I think best to leave as-is until we can figure out the right way forward for them.